lptools.cc

```diff
@@ -243,7 +243,10 @@ int main(int argc, char **argv) {
         auto partName = argv[2];
         auto dmState = android::dm::DeviceMapper::Instance().GetState(partName);
         if(dmState == android::dm::DmDeviceState::ACTIVE) {
-            android::fs_mgr::DestroyLogicalPartition(partName);
+            if (!android::fs_mgr::DestroyLogicalPartition(partName)) {
+                std::cerr << "Unable to unmap " << partName << std::endl;
```

Comment:
well, "remove" or "destroy" rather than "unmap"?

Reply:
unmap is the [command](https://github.com/phhusson/vendor_lptools/blob/8a9815341b2c1d73cc50d32f4f8fd2652d05851e/lptools.cc#L238) issued, but I'm happy to reroll it with the verb from the actual function name.

Reply:
Ah, I'm lost in my own code.
I guess you're wright.
I'll have to check again what DestroyLogicalPartition does, because... it sounds like... it destroys the partition -_-'

Reply:
It removes the mapping to the partition (e.g. /dev/block/dm-5), but it leaves the partition itself alone.